ucm2/sof-hda-dsp/HiFi.conf

```diff
@@ -84,6 +72,7 @@ SectionDevice."Speaker" {
 		PlaybackSwitch "Speaker Playback Switch"
 		PlaybackChannels "2"
 		PlaybackPriority "100"
+		JackHWMute "Headphone"
```

Comment:
This is PA or the UCM specification issue? The specification says this: _If **JackHWMute** is set, it indicates that when the jack is plugged in, the hardware automatically mutes some other device(s)._

And JackHWMute belongs to the JackControl which is not defined in Speaker device.

Reply:
hmm I don't know actually. With your ucm I was not seeing headset and speakers at the same time ever, so inserting headphones removed the speakers, thus I was not able to manually route to speakers when headset was connected. I was just diffing the differences and this was making it possible. So not sure is it ucm, pulseaudio, or ubuntu ui issue.

So I think this was in your original "Add initial support for generic SOF Skylake driver (DMIC)" import, but was changed during the rename, so actually don't know the originators thinking.

Reply:
My changes (trying to keep the one verb only) are in https://github.com/perexg/alsa-ucm-conf/commits/sof , if you like to test. I do tests with the latest alsa-lib and patched PA.